### ISSUE TITLE
cli_auth is never used?

### ISSUE THREAD

**Issue by iDVB** [c301/grunt-webstore-upload#12]
setting `cli_auth: true` does not seem to change anything. After reviewing `webstore_upload.js` it appears its never being used?

Future feature, just stub code now?

**Comment by pimterry** [comment]
Yes it, see https://github.com/c301/grunt-webstore-upload/blob/master/tasks/webstore_upload.js#L51, and https://github.com/c301/grunt-webstore-upload/commit/623a6682395b773c1c23a262f97eacdec6801740 for the full commit adding this functionality, back in January.
